packages/x-components/.eslintrc.js

```diff
@@ -8,7 +8,8 @@ module.exports = {
   rules: {
     'no-dupe-class-members': 'off',
     '@typescript-eslint/no-unused-vars-experimental': 'off',
-    'vue/require-default-prop': 'off'
+    'vue/require-default-prop': 'off',
+    '@typescript-eslint/explicit-function-return-type': 'off'
```

Comment:
In my opinion this rule is usually useful as it forces us to type functions so I wouldn't deactivate if for all the cases, unless there are a lot of places where the type is already missing.

Reply:
oh boy, dealing with eslint rules will be a problem of their own when we will update versions😅

I kind of agree with @alvarodE here

Reply:
The return type is already inferred by TS. I see it like, parameters need types because they are the input. But the output, the return type & value, I prefer TS gives it for me.

I am now against over-typescript 🤣